content/JENKINS/styles/assets/file-116-wait.gif

```diff
@@ -68,21 +68,6 @@
     <link rel="icon" type="image/x-icon" href="/s/en_GB/8100/5084f018d64a97dc638ca9a178856f851ea353ff/7/_/favicon.ico">
```

Comment:
shouldn't this whole file be deleted? it's a gif file not text

Reply:
I just deleted the <script> part from this gif and the other svg.  
All these files containing "Oops - an error has occurred" seem to have been generated from/thanks to an error, and could indeed be deleted.